scripts/run_notebooks/runner.py

```diff
@@ -25,27 +30,102 @@ def setup_logging() -> None:
     )
 
 
-def get_cwd_from_notebook_path(notebook_path: Path) -> str:
-    return str(notebook_path).rsplit("/", 1)[0]
+INJECTED_CODE = """
+import pymc as pm
+import arviz as az
+import xarray as xr
+import numpy as np
```

Comment:
can we isort these :)

Reply:
I've put in a file itself so it will go through the linter and formatting